node/index.js

```diff
@@ -0,0 +1,4 @@
+import { x } from y;
+
+console.log(x);
+console.log("Hell World")
```

Comment:
There is a typo here: "Hell World" should be "Hello World". Please update the string to fix the spelling.